python/gtsam/examples/DogLegOptimizerExample.py

```diff
@@ -0,0 +1,118 @@
+"""
```

Comment:
All these examples lost their git history? Can this be corrected (e.g. by saving backups, git mv, and then copy backups back?)

Reply:
Impossible to review as is. But, we did review them in a previous PR (I guess).

Reply:
Yes, I can mv cython here and overwrite :)

Reply:
Done